stock_move_line_auto_fill/README.rst

```diff
@@ -0,0 +1,143 @@
+possibility of transferring directly.
+
+Also this module adds a button in backorder confirmation wizard to auto
+complete to do quantities for products without lots.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Make sure you have selected the proper removal strategy on your product
+   categories.
+#. Configure the product on the page "Inventory", field "Tracking" with one of
+   these values: "By Unique Serial Number" or "By Lots" if you want autoassign
+   lots.
+#. Check in Operation type if you want auto assign quantities, by default if
+   you want assign quantities done to reserved quantities you must push the
+   button "Auto Fill" when the picking is in ready state and has move lines
+#. Check in Operation type if you do not want auto assign lots quantities with
+   "Avoid auto-assignment of lots".
+
+Usage
+=====
+
+Set options in operation types, you can set 'Auto fill operation' what make
+autoassignment button invisible and fill the quantities in operations for lines
+and 'Avoid Autoassignment Lots' what allow fill manually the lots quantities.
+
+Products without tracking lots
+==============================
+After confirming the picking, click on `Auto fill operations` button. The
+Operations matching the following conditions will be filled automatically:
+
+* The operation has not be processed (i.e `qty_done == 0`).
+* The operation has no package set (i.e `package_id` is empty).
+
+Product with lots
+=================
+#. Create an outgoing or an internal picking.
+#. Include one product with lots and with enough stock.
+#. Click on "Mark as Todo" button, and then on "Reserve".
+#. Clicking on the icon with the three items bullet list on the "Operations"
+   tab you will see that the quantities have been auto-assigned on the "Done"
+   column.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/stock-logistics-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/stock-logistics-workflow/issues/new?body=module:%20stock_move_line_auto_fill%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* ACSONE SA/NV
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* Zakaria Makrelouf (ACSONE SA/NV) <z.makrelouf@gmail.com>
+* Luis Triana (Jarsa Sistemas de S.A. de C.V.) <luis.triana@jarsa.com.mx>
+* `Tecnativa <https://www.tecnativa.com>`_:
+    * Pedro M. Baeza <pedro.baeza@tecnativa.com>
+    * Vicent Cubells <vicent.cubells@tecnativa.com>
+    * Sergio Teruel <sergio.teruel@tecnativa.com>
+    * David Vidal <david.vidal@tecnativa.com>
```

Comment:
As person who migrate this addon, I believe you can add your credit here.

Reply:
OK!